apps/src/templates/rubrics/EvidenceLevels.jsx

```diff
@@ -25,61 +17,20 @@ export default function EvidenceLevels({
     return newArray.sort((a, b) => b.understanding - a.understanding);
   };
   if (canProvideFeedback) {
```

Comment:
`canProvideFeedback` is false on the non-project level or when not viewing a student. We might have to pass in whether the user is a student or teacher here. It helps that students should be seeing `StudentRubricView` while teachers should be seeing `RubricContent`

Reply:
I need the face-palm emoji here... I knew that!  I ended up using `isStudent` from the student view which was easier than trying to navigate passing through a prop that uses the teacher indicator.